include/retdec/cpdetect/cptypes.h

```diff
@@ -175,16 +176,19 @@ struct ToolInformation
 	std::vector<DetectResult> detectedTools;        ///< detected tools (compilers, packers...)
 	std::vector<DetectLanguage> detectedLanguages;  ///< detected programming language(s)
 
-	bool entryPointOffset = false;   ///< @c false if file has no has no or invalid EP offset
-	unsigned long long epOffset;     ///< entry point offset
+	bool entryPointOffset = false;                  ///< @c false if file has no has no or invalid EP offset
+	unsigned long long epOffset;                    ///< entry point offset
 
-	bool entryPointAddress = false;  ///< @c false if file has no has no or invalid EP address
-	unsigned long long epAddress;    ///< entry point address
-	unsigned long long imageBase;    ///< image base address
+	bool entryPointAddress = false;                 ///< @c false if file has no has no or invalid EP address
+	unsigned long long epAddress;                   ///< entry point address
+	unsigned long long imageBase;                   ///< image base address
```

Comment:
Nitpick: Were these indentation changes necessary?

Reply:
I like when all the comments are on the same columns. I'll remove the changes if you don't

Reply:
I see. That's alright, you can keep the changes.